src/components/MainContainer.js

```diff
@@ -641,13 +587,20 @@ class MainContainer extends Component {
                                 dateRange={this.state.dateRange}
                                 onStatSliderChange={this.handleStatSliderChange}
                                 onDateSliderChange={this.handleDateSliderChange}
-                                // // onReprSliderChange={this.handleReprSliderChange}
-                            />
+                                 />
                             }
-                        </div>
+                        </Col>
+                    </Row>
+                </Content>
+
+                {/* MainChart Component */}
+                <Content style={{ background: 'white', padding: '50px 0' }}>
```

Comment:
Looks like you've been inline the styles. It's quick and easy to do this but one of the downsides is that it can't be reused.

I noticed `background: 'white', padding: '50px 0'` is used three times in this file. If you ever wanted to change this to `background: 'white', padding: '55px 0'`, you would have to find all the places that are using this style. Instead you can make a separate file that holds common styles.

it would looks something like

```
var commonStyles = {
  mainContainerStyle: {
    background: "white",
    padding: '50px 0',
  },
  someOtherStyle: {
    background: "blue"
  },
}
```

You would then import the `commonStyles` const on use it like...
`<Content style={commonStyles.mainContainerStyle}}>`